src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -529,4 +529,13 @@
   <data name="PreferIsKindFix" xml:space="preserve">
     <value>Use 'IsKind' instead of 'Kind'</value>
   </data>
+  <data name="MissingSourceGeneratorAttributeDescription" xml:space="preserve">
+    <value>'{0}' inherits from ISourceGenerator but does not have the [Generator] attribute</value>
```

Comment:
Looks like the resource analyzer is broken, nothing was raised to add the punctuation to the description.

Reply:
Nevermind, the issue is raised.

Reply:
Fixed, or at least ready to be flagged by the analyzer again :)